oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -342,6 +342,11 @@ receiver-async-profiler:
   default:
     # Used to manage the maximum size of the jfr file that can be received, the unit is Byte, default is 30M
     jfrMaxSize: ${SW_RECEIVER_ASYNC_PROFILER_JFR_MAX_SIZE:31457280}
+    # Used to determine whether to receive jfr in memory file or physical file mode
+    # Memory files have fewer local file system limitations, so they are by default. But it costs more memory.
+    # When users need to parse large JFR files(rare case), they could switch to file mode.
+    # For details, please refer to the comments of class AsyncProfilerModuleConfig
```

Comment:
This is end-user oriented, prefer duplicating the comment here instead of asking user to check the source code.